### ISSUE TITLE
Missing 'graphql-tag' module in 2.0.0-beta.5

### ISSUE THREAD

**Issue by loicplaire** [apollographql/apollo-server#1152]
I believe I found an issue in the latest beta (2.0.0-beta.5).

### How to reproduce

1. Checkout https://github.com/apollographql/graphql-server-example.
2. Run `npm i` and `node index.js`. 
3. You should get the following error:

```bash
module.js:549
    throw err;
    ^

Error: Cannot find module 'graphql-tag'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/git/graphql-server-example/node_modules/apollo-server/node_modules/apollo-server-core/dist/index.js:25:21)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

**Comment by eugene1g** [comment]
Pinging @evans as this bricks the release :)

A temp workaround is to install the dependency directly e.g. `yarn add graphql-tag`

**Comment by rashthedude** [comment]
Encountered the same exact problem.

**Comment by Ken123777** [comment]
Same problem for me too

**Comment by rookiedz** [comment]
Same problem.

**Comment by rodrigooler** [comment]
Same problem.

**Comment by sachinrajgire** [comment]
Just add
 npm install --save graphql-tag
It resolved my issue

**Comment by rashthedude** [comment]
@sachinrajgire it does but I reckon the underlying missing package should be added by apollo-server.

**Comment by evans** [comment]
Sorry about that! https://github.com/apollographql/apollo-server/pull/1125/commits/83c866e8d447b1510c268ccf6b592aa008d5b2d4 will be in the next release in a couple of hours

**Comment by Ken123777** [comment]
Thanks @evans

**Comment by evans** [comment]
We just published 2.0.0-beta.9, which includes graphql-tag ad a dependency
